pkg/client/request_test.go

```diff
@@ -748,6 +742,64 @@ func TestDoRequestNewWay(t *testing.T) {
 	}
 }
 
+func TestCheckRetryClosesBody(t *testing.T) {
```

Comment:
It's a bad name - I couldn't find a way to reach into the underlying default transport and determine that multiple TCP session have been opened.

----- Original Message -----
> > @@ -748,6 +742,64 @@ func TestDoRequestNewWay(t *testing.T) {
> >  	}
> >  }
> >  
> > +func TestCheckRetryClosesBody(t *testing.T) {
> 
> Maybe I'm dense, but how does this actually test that the client closes the
> body properly?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/6671/files#r28165287